velero-2-mixin/variables.libsonnet

```diff
@@ -0,0 +1,95 @@
+local g = import './g.libsonnet';
+local var = g.dashboard.variable;
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+
+// Generates chained variables to use on on all dashboards
+{
+  new(this, varMetric):
+    {
+      local filteringSelector = this.config.filteringSelector,
+      local groupLabels = this.config.groupLabels,
+      local instanceLabels = this.config.instanceLabels,
+      local topClusterSelector =
+        var.custom.new(
+          'top_cluster_count',
+          values=[2, 3, 6, 8, 10],
```

Comment:
Is there a field value that let's you set the default value. Right now it is 2. Not sure if that is a good default value for this integration or not.

Reply:
I cannot seem to find anything that would set a default value. I think 2 is fine here as it's been done in previous integrations.